kloppy/infra/serializers/event/statsbomb/helpers.py

```diff
@@ -14,7 +14,13 @@
 from kloppy.exceptions import DeserializationError
 
 
-def parse_str_ts(timestamp: str) -> float:
+def parse_football_clock_str_ts(timestamp: str) -> timedelta:
+    """Parse a mm:ss string football time into number of seconds."""
+    m, s = timestamp.split(":")
+    return timedelta(seconds=int(m) * 60 + float(s))
+
+
+def parse_str_ts(timestamp: str) -> timedelta:
```

Comment:
I would integrate it in a single method. You can count the number of splits to decide whether it is HH:mm:ss or mm:ss format string.